### DISCUSSION TITLE
select 选择器

### DISCUSSION THREAD

**Discussion by LunaticNote** [Tencent/tdesign#571] [Ideas]
最近在新的技术选择的时候看到了贵司的框架，感觉很适合我的使用场景，但是有一种遇见的select 选择器没有找到，在pc 和android 都遇到了这种需求，有其他的框会根据这个选择加载其中的信息，希望可以可以有类似的组件
![1733384869412](https://github.com/user-attachments/assets/dfbc9a5c-0ec1-4468-b1d2-813e5a1248b3)

希望这个项目越来越好

**Comment by uyarn** [comment]
这个可以用SelectInput自己组合下Table使用 SelectInput的panel是完全自定义的 https://tdesign.tencent.com/vue-next/components/select-input
